workers/unity/ProjectSettings/EditorSettings.asset

```diff
@@ -14,8 +14,9 @@ EditorSettings:
   m_EtcTextureFastCompressor: 1
   m_EtcTextureNormalCompressor: 2
   m_EtcTextureBestCompressor: 4
-  m_ProjectGenerationIncludedExtensions: txt;xml;fnt;cd
+  m_ProjectGenerationIncludedExtensions: txt;xml;fnt;cd;asmdef
```

Comment:
what are these changes

Reply:
Oh damn, slipped through. It's things I've been asking "what is that and why do I keep seeing it changed by Unity lately" but managed to avoid committing until now...

Reply:
it's still in here :)

Reply:
still here